Tasks/RunDistributedTests/README.md

```diff
@@ -1,2 +1,103 @@
-# Run Distributed Tests Task
+# Visual Studio Test using Test Agent Task
+
+###Overview
+The Visual Studio Test using Test Agent task should be used when you want to run tests on Test Agent machine(s). Given multiple machines in the Test Machine Group, the task can do parallel distributed runs of your tests. Parallelism is at the test assembly level. To use this task, it needs to be preceded with “Visual Studio Test Agent Deployment” task.
```

Comment:
The Visual Studio Test using Test Agent task should be used when you want to run tests on Test Agent machine(s). 

This seems to be a very less informative to a new user. Can we borrow a line or two from existing blogs to attract more users, on the benefits of doing remote testing as against local testing on build agent. and also a line to mention that this replaces TCTA as a way to do remote testing.